templates/tags.html

```diff
@@ -2,12 +2,33 @@
 
 {% block title %} &ndash; {{ _('Tags') }}{% endblock %}
 
+{% block meta %}
+  {% if PLUGINS and 'tag_cloud' in PLUGINS %}
+    <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/stylesheet/tag_cloud.css">
+  {% endif %}
+{% endblock %}
+
 {% block content %}
 <article class="single">
   <header>
     <h1 id="tags">{{ _('Tags') }}</h1>
   </header>
-  <div>
+
```

Comment:
This does not make sense in here, this template is for a list of tags, tag cloud is something for home pages.